cygnus-common/pom.xml

```diff
@@ -99,7 +99,7 @@
         <dependency>
             <groupId>org.apache.hive</groupId>
             <artifactId>hive-jdbc</artifactId>
-            <version>2.3.4</version>
+            <version>3.1.2</version>
```

Comment:
This is an update in a common library.

No problem in doing so if it doesn't cause any problem in existing Cygnus bundles (I guess that cygnus-ngsi uses it, not sure in the case of cygnus-twitter), but the upgrade should be mentioned in the CHANGES_NEXT_RELEASE file.

Btw, the creation of the new cygnus-ngsi-ld bundle should be also mentioned in the CHANGEX_NEXT_RELEASE file.

Reply:
Done 078980f